lib/ansible/modules/user.py

```diff
@@ -3317,6 +3354,7 @@ def main():
             password_expire_account_disable=dict(type='int', no_log=False),
             uid_min=dict(type='int'),
             uid_max=dict(type='int'),
+            unsafe_writes=dict(type='bool', default=False, fallback=(env_fallback, ['ANSIBLE_UNSAFE_WRITES'])),
```

Comment:
`env_fallback` is undefined.

Reply:
yes, I am writing test for Alpine user creation and enabling user tests for alpine.